.github/workflows/tests.yml

```diff
@@ -19,6 +19,7 @@ jobs:
             ${{ runner.os }}-node-
       - run: npm install
       - run: npm run build
+      - run: npm run docs
```

Comment:
@jonathanKingston this will just generate the output, and will fail on errors. For now this gets the flow in place and we can follow with configuring the deployment - luckily I've been through that a few times already with this tool, so I know all the steps needed